### PR TITLE
Use `bedrock-app-identity`.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -23,8 +24,12 @@ bedrock.events.on('bedrock.init', async () => {
     handler({meter} = {}) {
       // use configured meter usage reporter as service ID for tests
       const clientName = mockData.productIdMap.get(meter.product.id);
-      meter.serviceId = bedrock.config['meter-usage-reporter']
-        .clients[clientName].id;
+      const serviceIdentites = getServiceIdentities();
+      const serviceIdentity = serviceIdentites.get(clientName);
+      if(!serviceIdentity) {
+        throw new Error(`Could not find identity: "${clientName}"`);
```
Comment (dlongley) [inline test/test.js:30]:
```suggestion
        throw new Error(`Could not find identity "${clientName}".`);
```

File: lib/config.js [hunk 2]
```diff
@@ -21,17 +21,11 @@ cfg.storageCost = {
...
-    type: 'Ed25519VerificationKey2020',
-    publicKeyMultibase: 'z6MkgwieJDAgdUQyD17L6TsGJon57jt3yxZrtw7rg3tJ5orH',
-    privateKeyMultibase: 'zrv4SV17sr6L2Ymyyu3Uj7cvJm3qvmMKbzW1wwJkpuGiQj5' +
-      'XqQeF5nmtwoP86VH7zzT2v4uS921gJzbztbdbCakwd8D'
-  }
+// `bedrock-app-identity` so it doesn't have to be set here
+config['app-identity'].seeds.services.edv = {
+  id: 'did:key:z6MkhNyDoLpNcPv5grXoJSJVJjvApd46JU5nPL6cwi88caYW',
+  seedMultibase: 'z1AgcCz4zGY5P3covUxqpaGTVs6U12H5aWH1FdyVABCwzkw',
+  serviceType: 'edv'
```
Comment (dlongley) [inline lib/config.js:30]: Do we need `serviceType` here?

**Review by dlongley — APPROVED** [review]
Thanks!

File: lib/config.js (hunk 2 above)
Comment (gannan08) [reply lib/config.js:30]: Yes, the name of the key may not equal the `serviceType` in some cases.

**Pull request merged**